src/applications/mhv-medical-records/util/pdfHelpers/blueButton.js

```diff
@@ -53,27 +53,26 @@ export const generateBlueButtonData = (
       `Showing ${labsAndTests?.length} records from newest to oldest`,
     ],
     selected: recordFilter.includes('labTests'),
-    records:
-      isArrayAndHasItems(labsAndTests) && labsAndTests.length > 0
-        ? labsAndTests.map(record => {
-            const title = record.name;
-            let content;
+    records: isArrayAndHasItems(labsAndTests)
```

Comment:
Removed labsAndTests.length > 0 since isArrayAndHasItems(labsAndTests) already performs the length check.